### PR TITLE
[special] migrate log1p, sinc, round to special namespace

### PR DESCRIPTION
**Pull request by kshitij12345**
Reference : #50345

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -7846,23 +7825,8 @@ def merge_dicts(*dicts):
...
-
-Example::
-
-    >>> a = torch.randn(4)
-    >>> a
-    tensor([ 0.9920,  0.6077,  0.9734, -1.0362])
-    >>> torch.round(a)
-    tensor([ 1.,  1.,  1., -1.])
-""".format(**common_args))
+Alias for :func:`torch.special.round`.
```
Comment (mruberry) [inline torch/_torch_docs.py:7828]: Let's keep torch.round the source of truth to be most consistent with the Python Array API (https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html?highlight=round#round-x) -- is our round equivalent to their round, by the way?

We can alias torch.special.round to torch.round

File: torch/_torch_docs.py [hunk 2]
```diff
@@ -4660,30 +4660,9 @@ def merge_dicts(*dicts):
...
-
-Example::
-
-    >>> a = torch.randn(5)
-    >>> a
-    tensor([-1.0090, -0.9923,  1.0249, -0.5372,  0.2492])
-    >>> torch.log1p(a)
-    tensor([    nan, -4.8653,  0.7055, -0.7705,  0.2225])
-""".format(**common_args))
+Alias for :func:`torch.special.log1p`.
```
Comment (mruberry) [inline torch/_torch_docs.py:4664]: Let's alias torch.special.log1p to torch.log1p for Python Array API consistency (see https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html?highlight=log1p#log1p-x).

**Review by mruberry — COMMENTED** [review]
This looks good but we should keep the docs for round and log1p in `torch` namespace for Python Array API consistency.

File: torch/_torch_docs.py (hunk 1 above)
Comment (kshitij12345) [reply torch/_torch_docs.py:7828]: Makes sense. Will keep the Array API in mind going forward. Thanks!

> is our round equivalent to their round, by the way?

Will confirm that!

**Comment by kshitij12345** [comment]
@mruberry this should be ready given that the CI passes. Thanks :)!

**Comment by mruberry** [comment]
It looks like only a few CI jobs ran, maybe this needs a rebase? cc @samestep -- do you know what's up with that?

**Review by mruberry — APPROVED** [review]
CI looks good; thanks @kshitij12345!

**Pull request closed without merging**